bakery/baking_usage_examples/RB_1_qubit/RB_utils.py

```diff
@@ -0,0 +1,110 @@
+import pandas as pd
+from bakery.bakery import *
+from qm.qua import *
+
+c1_ops = [  # Clifford operations
+    ('I',),
+    ('X',),
+    ('Y',),
+    ('Y', 'X'),
+
+    ('X/2', 'Y/2'),
+    ('X/2', '-Y/2'),
+    ('-X/2', 'Y/2'),
+    ('-X/2', '-Y/2'),
+    ('Y/2', 'X/2'),
+    ('Y/2', '-X/2'),
+    ('-Y/2', 'X/2'),
+    ('-Y/2', '-X/2'),
+
+    ('X/2',),
+    ('-X/2',),
+    ('Y/2',),
+    ('-Y/2',),
+    ('-X/2', 'Y/2', 'X/2'),
+    ('-X/2', '-Y/2', 'X/2'),
+
+    ('X', 'Y/2'),
+    ('X', '-Y/2'),
+    ('Y', 'X/2'),
+    ('Y', '-X/2'),
+    ('X/2', 'Y/2', 'X/2'),
+    ('-X/2', 'Y/2', '-X/2'),
+
+]
+
+# Cayley table corresponding to above Clifford group structure
+
+c1_table = pd.read_csv('c1_cayley_table.csv').to_numpy()[:, 1:]
+
+
+class RB_one_qubit:
+    def __init__(self, config: dict, d_max: int, K: int, qubit: str, cliffords: List = None,
+                 cayley_table: np.ndarray = None):
+        """
+        Class to retrieve easily baked RB sequences and their inverse operations
+        :param config Configuration file
+        :param d_max Maximum length of desired RB sequence
+        :param K Number of RB sequences to be generated
+        :param qubit Name of the quantum element designating the qubit
+        :param cliffords List of Clifford operations chosen, by default c1_ops defined above
+        :param cayley_table Table indicating group structure associated to Cliffords provided
+        """
+
+        if cliffords is None:
```

Comment:
Because Cliffords should be indexed according to what is inside the Cayley table, the two inputs should be related. So I can remove the possibility for the user to tune up its set of Cliffords, but I cannot give access to the Clifford set without knowing how to navigate through the Cayley table.